package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@herbsjs/herbsshelf",
-  "version": "3.1.1",
+  "version": "4.0.0",
```

Comment:
did you change it manually or it was changed by the CI? 
since it is not a breaking change, the version number should be 3.*

Reply:
Hi @dalsoft, I changed the method renderShelfHtml method adding entities parameter. It will be broke the projects that use all parameters. Make sense?

Reply:
@dalssoft I make the change manually. I will use semantic-release to indicate a breaking change.

Reply:
Regardless of the discussion here about the `renderShelfHTML` method, we shouldn't change the version manually and let CI/CD do it automatically. Just use `git cz` for that.